NOnion/NOnion.fsproj

```diff
@@ -88,7 +89,7 @@
 
   <Target Name="CopyProjectReferencesToPackage" DependsOnTargets="ResolveReferences">
     <ItemGroup>
-      <BuildOutputInPackage Include="@(ReferenceCopyLocalPaths->WithMetadataValue('ReferenceSourceTarget', 'ProjectReference'))" />
```

Comment:
@aarani this change seems unrelated to the commit at hand

Reply:
honestly couldn't find the commit this was in.

Reply:
git blame?